Problem2.java

```diff
@@ -0,0 +1,52 @@
+// Time Complexity : O(n2)
```

Comment:
You are missing the time complexity for sorting. It will be O(nlogn) + O(n^2) and asymptotically it will O(n^2).